tests/rkt_fetch_test.go

```diff
@@ -147,10 +147,6 @@ func TestImplicitFetch(t *testing.T) {
 			t.Fatalf("Cannot exec rkt: %v", err)
 		}
 
-		if err := child.Expect(foundMsg); err != nil {
-			t.Fatalf("Expected %q but not found: %v", foundMsg, err)
-		}
-
```

Comment:
I understand we don't get the found message because we don't get any metadata when using docker images and that forces the re-download.

We should modify the test comment and write something like this instead of not expecting anything:

```
if err := expectWithOutput(child, foundMsg); err == nil {
    t.Fatalf("%q should not be found", foundMsg)
}
```